control/passivity.py

```diff
@@ -0,0 +1,72 @@
+'''
+Author: Mark Yeatman  
+Date: May 15, 2022
+'''
+
+import numpy as np
+
+try:
+    import cvxopt as cvx
+except ImportError as e:
+    cvx = None
+
+lmi_epsilon = 1e-12
+
+def is_passive(sys):
+    '''
+    Indicates if a linear time invariant system is passive
```

Comment:
Please use [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) to indicate the input parameter and return type.